doc/releases/changelog-dev.md

```diff
@@ -39,6 +39,8 @@
 
 <h3>Internal changes ⚙️</h3>
 
+* When capture is enabled, `qjit(autograph=True)` will use capture autograph instead of catalyst autograph.
```

Comment:
```suggestion
* When capture is enabled, `qjit(autograph=True)` will use capture autograph instead of catalyst autograph.
  [(#1960)](https://github.com/PennyLaneAI/catalyst/pull/1960)
```